### ISSUE TITLE
NTR: Latrine used for storage, observed

### ISSUE THREAD

**Issue by SheenaTomko** [VEuPathDB-ontology/VEuPathDB-ontology#512]
For new term requests, please provide the following information:
**Term Label** [required] Latrine used for storage, observed

**Synonyms** [optional]

**Definition** [required except for organism name]
The definition should be understandable even for non-specialists. Include a PubMed ID or DOI to refer to any relevant article that provides additional information about the suggested term.

capture whether the interviewer observed the latrine and found it used for storage.

**Suggested Parent Term** [required]
Please look in the ontology hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/eupath) or choose one from following list: 
- [ ] Material (independent of other entities and can be maintained through time, such as person, cell, sample)
- [ ] Process (has temporal proper parts and for some time depends on some material entity, such as life cycle, cell division, heart beating, sleeping)
- [ ] Information (generically dependent on some artifact and stands in relation of aboutness to some entity, such as data, identifier, document)
- [ ] Characteristic/Quality (inheres in or is borne by other entities, such as height, color, disease)

Information 

**Examples** [optional]

**Contributor** [optional]
If you would like to be a contributor of the term or anyone you think that contributes to the term, such as label, definition, please provide the name and ORCID ID.

**Requester** [optional]
Please indicate which resource/database/project/ request the term.
ClinEpiDB

**Comment by johnwjudkins** [comment]
@SheenaTomko 
EUPATH:0048312 ( <--- is this ID correct? )
Label: indicator of latrine used for storage
Definition: Information on disposal of human excreta material that specifies whether a latrine is used for storage.
Parent: Information on disposal of human excreta material (EUPATH:0000596, same parent as in ClinEpi)

**Comment by SheenaTomko** [comment]
@johnwjudkins, yes the ID is correct. The definition however doesn't indicate that this is something confirmed by the interviewer. We have another term (Storage, multifilter value for Latrine use EUPATH_0048009) where it's use for storage was reported by the interviewee. Here we want to make it clear that this was confirmed by an outside observer

**Comment by johnwjudkins** [comment]
Can we say it was observed by an [investigation agent](http://purl.obolibrary.org/obo/OBI_0000202), or do you think it needs  to be more specific?

Definition: Information on disposal of human excreta material that specifies whether a latrine was observed by an investigation agent as being used for storage.

**Comment by SheenaTomko** [comment]
@johnwjudkins that seems like it would work
